src/unleash.ts

```diff
@@ -133,6 +133,14 @@ export class Unleash extends EventEmitter {
             this.emit('warn', msg);
         });
 
+        this.repository.on('serveralive', () => {
+            this.emit('serveralive');
+        });
+
+        this.repository.on('repochange', data => {
+            this.emit('repochange', data);
```

Comment:
I think i would prefer the event name to simply be `update`.

Reply:
Agreed. I will change it tomorrow morning. In retrospect I am not too keen on the”serveralive” either. It is not what it represents. How about “responding”, or  “connected”?

Reply:
I was thinking the same thing, but I did not have any better suggestion. What do you intend to use serveralive for?

What about: 
- `changed` - when toggle configuration are updated
- `unchanged` - 304 response from server, nothing new. 

(The Java client just returns a FearureToggle response with the status code as data. I'm not convinced that's any better.)

Reply:
I like that. Let us go with `changed` and `unchanged`.

The intended use for `serveralive` is to let the client know if it was able to reach the server. By combining `changed` and `unchanged` you would get the same effect.

So now you can control what to do if unleash crashes or something else goes wrong. In this case, fallback to fa falsy response. That is kinda cool:
```
let connected = false
const instance = initialize({...})

instance.on('unchanged', () => {
  connected = true
})

instance.on('changed', () => {
  connected = true
})

instance.on('error', () => {
  connected = false
})

function isEnabledWithFallback(toggle, context, fallback) {
  if (!connected) { return false }  // fallback to false if connection to unleash somehow goes wrong
  return isEnabled(toggle, context, fallback)
}

module.exports = {
  isEnabled: isEnabledWithFallback
} 
```